static/src/javascripts/projects/commercial/modules/dfp/Advert.ts

```diff
@@ -84,21 +121,28 @@ class Advert {
 		adSlotNode: HTMLElement,
 		additionalSizeMapping: SizeMapping = {},
 	) {
+		// Try to used size mappings if available
 		const defaultSizeMappingForSlot = adSlotNode.dataset.name
 			? getSlotSizeMapping(adSlotNode.dataset.name)
 			: {};
 
-		const sizeMapping = mergeSizeMappings(
+		let sizeMapping = mergeSizeMappings(
 			defaultSizeMappingForSlot,
 			additionalSizeMapping,
 		);
 
+		// If the size mapping is empty, use the data attributes to create a size mapping, this is used on some interactives
```

Comment:
Could we add an example link to an article where this issues surfaces pls. Thanks